integration/integration_test.go

```diff
@@ -516,7 +585,7 @@ func TestSeccompadvisor(t *testing.T) {
 		waitUntilTestPodReadyCommand(ns),
 		{
 			name:           "Run seccomp-advisor gadget",
-			cmd:            fmt.Sprintf("id=$($KUBECTL_GADGET seccomp-advisor start -n %s -p test-pod); sleep 30; $KUBECTL_GADGET seccomp-advisor stop $id", ns),
+			cmd:            fmt.Sprintf("id=$($KUBECTL_GADGET advisor seccomp-policy start -n %s -p test-pod); sleep 30; $KUBECTL_GADGET advisor seccomp stop $id", ns),
```

Comment:
```suggestion
			cmd:            fmt.Sprintf("id=$($KUBECTL_GADGET advisor seccomp-policy start -n %s -p test-pod); sleep 30; $KUBECTL_GADGET advisor seccomp-policy stop $id", ns),
```

Reply:
I should have used `s///g` and not `s///` :sweat_smile:!